### DISCUSSION TITLE
What is the difference between ScreenSpaceDrawQuad and the conservative one

### DISCUSSION THREAD

**Discussion by SanicBTW** [ppy/osu-framework#6428] [Q&A] [answered]
Since I started writing Draw Nodes I usually use ScreenSpaceDrawQuad to use the draw quad of the sprite whose Draw Node I'm changing but I wonder what ConservativeScreenSpaceDrawQuad does, I see it's used on some of the Draw Node functions but that's all I've researched so far.

**Comment by smoogipoo** [comment, marked as answer]
Should probably be documented better (I've done this in the past in combination with other changes that haven't been complete yet...), but it's the "inner" masking SSDQ that excludes corner radius, border, and masking smoothness effects.

It's primarily used in the front-to-back pre-pass which populates the depth buffer by drawing quads under where each `Sprite` in the game would go to save on fill rate (i.e. ["Early-Z"](https://developer.arm.com/documentation/102224/0200/Early-Z)).

**Reply by SanicBTW** [reply]
Ohh I kind of understand it now, tysm!!
